lib/KBreadcrumbs.vue

```diff
@@ -1,317 +1,157 @@
 <template>
 
-  <div
-    v-show="showSingleItem || crumbs.length > 1"
-    :class="{ 'breadcrumbs-collapsed': collapsedCrumbs.length }"
-  >
-    <nav
-      class="breadcrumbs"
-      v-bind="$attrs"
-      :aria-label="$attrs.ariaLabel"
+  <div class="breadcrumbs">
+    <KListWithOverflow
+      overflowDirection="start"
+      :items="preparedCrumbs"
     >
-      <div
-        v-show="collapsedCrumbs.length"
-        class="breadcrumbs-dropdown-wrapper"
-      >
+      <!-- Render individual breadcrumb items -->
+      <template #item="{ item,index }">
+        <li>
```

Comment:
We will also need to set the maxWidth that we set to the spans here to the li, for it to know what its its own max width, instead of getting it from the listOverflow container.